src/main/webapp/index.html

```diff
@@ -21,24 +25,55 @@
-          <p>Enter the maximum number of comments to be analyzed: </p>
-          <input type="range" id="num-comments-input" name="num-comments-input" min="100" max="1000" value = "100" step="10"/>
-          <small id="slider-output" name = "slider-output">100</small>
-          </br>
-          <input type="submit" id="submit-btn"/>
-          </br>
-        </form>
+      </li>
+    </ul>
+    <div id="youtube-data">
+      <div class="form-group">
+        <nav>
+          <div class="nav nav-tabs" id="nav-tab" role="tablist">
+            <a class="nav-item nav-link active" id="nav-keyword-tab" data-toggle="tab" href="#nav-keyword" role="tab" aria-controls="nav-keyword" aria-selected="true">Keyword Search</a>
+            <a class="nav-item nav-link" id="nav-videolink-tab" data-toggle="tab" href="#nav-videolink" role="tab" aria-controls="nav-videolink" aria-selected="false">Video Link</a>
+          </div>
+        </nav>
+        <div class="tab-content" id="nav-tabContent">
+          <div class="tab-pane fade show active" id="nav-keyword" role="tabpanel" aria-labelledby="nav-keyword-tab">
+            <br>
+            <p>Enter a YouTube video keyword:</p>
+            <input type="text" id="search-input">
+            <input name="Search" type="submit" value="Search!" id="submit-button" onclick="getVideoResults()"/>
+            <p>Maximum number of comments: </p>
+            <input type="range" id="num-comments-input" name="num-comments-input" min="100" max="1000" value = "100" step="10"/>
+            <small id="slider-output" name = "slider-output">100</small>
+            </br>
+            <div id="video-results" style="margin-top:20px">
+              <div class="row">
+                <div class="col-4">
+                  <div id= "tab_list" class="list-group" role="tablist" ></div>
+                </div>
+                <div class="col-8">
+                  <div id= "info_list" class="tab-content" id="nav-tabContent"></div>
+                </div>
+              </div>
+            </div>
+            </br>
+          </div>
+          <div class="tab-pane fade" id="nav-videolink" role="tabpanel" aria-labelledby="nav-videolink-tab">
+            <br>
+            <p>Enter a YouTube link:</p>
+            <input id="link-input" name="link-input" required pattern="https?://www.youtube.com/watch\?v=.+"/>
+            <small id="url-help" class="form-text text-muted">Format: https://youtube.com/watch?v=...</small>
+            </br>
+            <button type="submit" class="btn btn-primary submitBtn" id="submit-btn" onClick="updateUIWithVideoContext();">Enter</button>
+            <br>
+            <p>Maximum number of comments: </p>
+            <input type="range" id="num-comments-input" name="num-comments-input" min="100" max="1000" value = "100" step="10"/>
+            <small id="slider-output" name = "slider-output">100</small>
```

Comment:
Once the element has been moved so it's not part of the tab, delete this repeat element as it does nothing.